index.html

```diff
@@ -2374,21 +2394,6 @@ <h3>
 consumption rules</a> in the [[INFRA]] specification.
         </p>
 
-        <p>
-The value of the <code>@context</code> object member MUST be ignored as this is
-reserved for JSON-LD consumers.
-        </p>
-
-        <p>
-Unknown object member names MUST be ignored as unknown properties.
```

Comment:
Please replace the deleted text above with the text of the resolution https://www.w3.org/2019/did-wg/Meetings/Minutes/2020-11-03-did#resolution2 : "Unrecognized properties MUST be preserved."  If this is not the right place, then put it somewhere else.  But the text of the approved resolution needs to normatively appear in the spec.

Reply:
I think the spirit of that resolution may be captured by this text, which is proposed above:
```
Consumers MUST add all properties that do not have explicit processing rules
for the representation being consumed to the abstract data model using only the
representation's generic type processing rules. Producers MUST serialize all
properties in the abstract data model that do not have explicit processing
rules for the representation being produced using only the representation's
generic type processing rules.
```

Reply:
I believe this suggestion addresses your concerns: https://github.com/w3c/did-core/pull/454/#discussion_r525329052